examples/gateway/credit_card_payment.php

```diff
@@ -53,7 +48,12 @@
     $ccDetails->setExpirationMonth('05');
     $ccDetails->setExpirationYear('2029');
     $ccDetails->setCvv(123);//do not store nor log the CVV
-    $creditCardPayment->setCreditCardDetails($ccDetails);
+
+    //Set this flag to true when you do not want to capture the transaction amount immediately, but only validate the
+    // payment details and block (reserve) the amount. The capture of a preauth-only transaction can be performed with
+    // the CapturePayment class.
+    $ccDetails->setIsPreAuth(false);
```

Comment:
Why is is_preauth in the payment details entity now? Is it because some payment methods like PIX do not support preauth only? I think it's best to mirror the api implementation in the SDK, and it something's not supported, the API response will tell you.

Reply:
moved back to Payment/Signup classes